### DISCUSSION TITLE
Is it possible to forbid the resizing of the window?

### DISCUSSION THREAD

**Discussion by ghost** [TomSchimansky/CustomTkinter#712] [Q&A]
Hello, is it possible to prevent the user from resizing the window (like with tkinter): window.resizable(height=False, width=False) ?

**Comment by TomSchimansky** [comment]
It's possible.

**Reply by ghost** [reply]
And how, I can't find the answer :/

**Comment by Akascape** [comment]
@LucasAubrt Just use `window.resizable(height=False, width=False)` for the CTk window.

Another method will be:
```python
window.maxsize(width, height) # put the CTk window's height/width
window.minsize(width, height)
```
Most of the tkinter methods like these are compatible with customtkinter.
